### PR TITLE
Add event schema for incident-triggered trigger

### PR DESCRIPTION
**Pull request by impl**

### CODE REVIEW

File: triggers/incident-triggered/event.schema.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      "description": "The account-unique incident number"
+    },
+    "title": {
+      "type": "string",
+      "description": "A succinct description of the incident"
+    },
+    "urgency": {
+      "type": "string",
+      "description": "The incident's urgency",
+      "enum": ["high", "low"]
```
Comment (woodburndesigns) [inline triggers/incident-triggered/event.schema.json:20]: We'll have to make sure @sprokusk can render this.

File: triggers/incident-triggered/event.schema.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The incident's unique identifier"
+    },
+    "incidentNumber": {
+      "type": "integer",
```
Comment (woodburndesigns) [inline triggers/incident-triggered/event.schema.json:10]: This was changed from a string in the readme. This intentional?

**Review by woodburndesigns — APPROVED** [review]
Approved with minor comments.

File: triggers/incident-triggered/event.schema.json (hunk 1 above)
Comment (impl) [reply triggers/incident-triggered/event.schema.json:20]: It's OK if it doesn't get rendered for now - it's additive in terms of documentation information anyway.

File: triggers/incident-triggered/event.schema.json (hunk 2 above)
Comment (impl) [reply triggers/incident-triggered/event.schema.json:10]: It is indeed an integer in their API response. Must have been a typo in the README.

**Pull request merged**